### PR TITLE
Redirect from `/` to `/space`.

### PR DESCRIPTION
**Pull request by fisx**
- Export `redirect` from `Frontend.Core`.
- Add instance for `ToHtml ()`.
- Add path `Top -> "/"`.
- Add route `/` redirecting to `/space`.

### CODE REVIEW

File: src/Frontend.hs [hunk 1]
```diff
@@ -69,6 +70,7 @@ aulaTop (Nat runAction) =
   :<|> (\req cont -> getSamplesPath >>= \path ->
           waiServeDirectory path req cont)
   :<|> waiServeDirectory (Config.config ^. htmlStatic)
+  :<|> redirect "/space"
```
Comment (andorp) [inline src/Frontend.hs]: This should be changed to `/login` in the future.

File: src/Frontend/Core.hs [hunk 2]
```diff
@@ -59,6 +60,11 @@ instance (ToHtml (HtmlT Identity ())) where
     toHtmlRaw = toHtml
     toHtml = HtmlT . return . runIdentity . runHtmlT
 
+-- | FIXME: Could this be a PR for lucid?
+instance (ToHtml ()) where
```
Comment (andorp) [inline src/Frontend/Core.hs]: Extra parenthesis.

**Comment by andorp** [comment]
LGTM

File: src/Frontend.hs (hunk 1 above)
Comment (fisx) [inline src/Frontend.hs]: I disagree.  If you try to hit a page and don't have access, you should be redirected to `/login`.  If you are loading `/`, you should be pointed to some page that is most likely to be usedful directly **after** login.  Even better than `/space` would be the current year's class idea space for students, the admin workbench for admins, and so on.  but this is still sci-fi...

Comment (andorp) [inline src/Frontend.hs]: Got your point. It is valid.

**Pull request merged**